sdk/template/template/package.json

```diff
@@ -78,7 +78,8 @@
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
-    "@azure/core-http": "^2.0.0",
+    "@azure/core-client": "^1.4.0",
+    "@azure/core-rest-pipeline": "^1.4.0",
```

Comment:
Did we have instructions anywhere on what versions to use here? From what I can see in other packages, we are using ^1.0.0 for core-client and ^1.1.0 for core-rest-pipeline

Reply:
There are no specific instructions on what versions to use on the migration guide, I use the latest here.

Reply:
isn't rush supposed to complain about inconsistent deps?

Reply:
![image](https://user-images.githubusercontent.com/10452642/150864388-3e2c8316-fce3-4c7f-aaf0-2b0b77c32d47.png)
We removed that flag :)

https://github.com/Azure/azure-sdk-for-js/pull/17587